### ISSUE TITLE
a11y: Can't access with keyboard to the "Edit..." button in "Mumble server connect" popup

### ISSUE THREAD

**Issue by zamentur** [mumble-voip/mumble#3410]
![bug_edit_mumble](https://user-images.githubusercontent.com/4080016/40569174-ba6de9c4-607f-11e8-92fd-8b9ff4f87506.png)

It seems on mumble 1.2.19-1 it's impossible to use the keyboard to open the "Edit..." popup. The tab can be used to navigate on the "Edit..." button, but when i press Enter it run the connexion action and don't display the "Edit..." popup.

I have a blind friend and she take long time with the mouse to succeed to open this popup...

Have you an idea if it's very complicated to fix it ?

**Comment by zamentur** [comment]
May be it's here:
https://github.com/mumble-voip/mumble/blob/master/src/mumble/ConnectDialog.cpp#L966

**Comment by tapper82** [comment]
I have same on windows 10 using the nvda screen reader

**Comment by alexarnaud** [comment]
@zamentur I don't know how to have access to the edit button, it's grayed on my computer.

See https://stackoverflow.com/questions/11887938/how-to-make-a-qpushbutton-pressable-for-enter-key to propose a fix.

Best regards,
Alex.

**Comment by davidebeatrici** [comment]
@alexarnaud `AutoDefault` is currently implicitly set to `false`:
https://github.com/mumble-voip/mumble/blob/2ad8c65165441c8250c7b5327e9cd1a858d833cc/src/mumble/ConnectDialog.cpp#L969

I'm going to create a pull request to change it to `true`, thank you for your input!

**Comment by hacst** [comment]
You can trigger the buttons using space. At least that seem to work for me on linux.

**Comment by Krzmbrzl** [comment]
To sum it up: The edit button is greyed out until you have selected one of your favorite servers (use arrow-keys to do that). Then you can trigger the edit button by using Alt+E.

If this doesn't work on your machine, let me know, but for now I assume this problem to be "fixed".

**Comment by zamentur** [comment]
Alt+E doesn't work with 1.2.19-1ubuntu1 . I pressed Alt + E and nothing happen, no edit pop up appears.

The shortcuts displayed in french are not correct, and we have twice the letter "A" underlined (see screen in initial bug report).

About the "space" tips, it works indeed. But i don't know how a blind-people, who browse by keyboard, could imagine they need to press "space" instead of "Enter". This behaviour is not a common behaviour (i think).

However, mumble-web accessibility may be better, and i probably advice blind people i know to use it instead.

**Comment by Krzmbrzl** [comment]
Could you try with Mumble 1.3.0 please?

For me Alt+C was used twice but by repeatedly using that shortcut, I was able to switch between the 2 buttons. Once I had the proper one selected, I can activate it via enter.
